beetsplug/replaygain.py

```diff
@@ -875,7 +880,7 @@ def _set_file(self):
         # Set a new file on the filesrc element, can only be done in the
         # READY state
         self._src.set_state(self.Gst.State.READY)
-        self._src.set_property("location", py3_path(syspath(self._file.path)))
+        self._src.set_property("location", py3_path(syspath(self._file)))
```

Comment:
It's also nice that `_file` now actually contains a file rather than an Item object.